test/integration/timelock.ts

```diff
@@ -53,40 +31,56 @@ describe("Execute with timelock", () => {
     stakingRewardsDistribution.transferOwnership(timelock.address);
   });
 
-  it("transaction should be executed before eta, withing grace period", async () => {
+  it("transaction should be executed after eta, within grace period", async () => {
```

Comment:
Once we'll nail down the naming, make sure to change the tests according to all the new names